woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -133,6 +133,11 @@ def test_unknown_inference(strings):
             inferred_type = ww.type_system.infer_logical_type(series.astype(dtype))
             assert isinstance(inferred_type, Unknown)
 
+            # verify transform works and column is still null
+            transform = inferred_type.transform(series.astype(dtype))
```

Comment:
I think that the `series.astype(dtype)` will be redundant -- `LogicalType.transform` will perform the astype call if needed